node/lib/openshift-origin-node/model/v2_cart_model.rb

```diff
@@ -400,6 +400,9 @@ def post_configure(cartridge_name)
           if cartridge.name == primary_cartridge.name and empty_repository?
             output << "CLIENT_MESSAGE: An empty Git repository has been created for your application.  Use 'git push' to add your code."
           else
+            if @state.value == State::STARTED
+              output << stop_cartridge(cartridge, user_initiated: true)
+            end
             output << start_cartridge('start', cartridge, user_initiated: true)
```

Comment:
@tiwillia  I see your point. Looked at the code and it seems like during post-receive step, the start gear process does use start cartridge as well. So I dont think we miss anything if we skip the second start process.